xAPI.md

```diff
@@ -3225,8 +3225,14 @@ Statements are filtered.
 * Activity Objects contain Language Map Objects for name, description and interaction components. 
 The LRS MUST return only one language in each of these maps. 
 
-* Verb Objects contain Language Map Objects for Display. 
-The LRS SHOULD* return only one language in this map. 
+* The LRS MAY maintain canonical versions of language maps for any language map used in an object
```

Comment:
What's the specific objection? How about "an object identified by an IRI"?

I'm going to make that change because I like it better anyway; please comment if it doesn't address your concerns.